catalogue/mpi/fhir-patient/api.yml

```
@@ -2,6 +2,8 @@ title: FHIR Patient
 description: Query for patients identifiers matched across multiple systems 
 type: REST/FHIR
 owner: NWIS - Ecosystem Team
+status: experimental
+access: public
```

I know the sandbox is public but I think if this experimental API went live it would have to be restricted as it's sensitive data
```suggestion
access: restricted
```